pages/training/modules/views/index.jsx

```diff
@@ -11,6 +11,7 @@ export default function Type() {
         title={<Snippet isExemption={model.isExemption}>title</Snippet>}
         subtitle={`${profile.firstName} ${profile.lastName}`}
       />
+      <p><Snippet>hint</Snippet></p>
```

Comment:
shouldn't it be `{hint}` ?

Reply:
@kiran-varma-home-office, just hint will only work

Reply:
@kiran-varma-home-office Snippet takes it's children as a string and looks up that snippet. so `<Snippet>hint</Snippet>` is correct.